bangazonapi/views/product.py

```diff
@@ -271,6 +272,14 @@ def sold_filter(product):
                 return False
 
             products = filter(sold_filter, products)
+        
+        if min_price is not None:
+            def price_filter(product):
+                if product.price > int(min_price):
```

Comment:
what if the price has cents?

(this is the ticket that my comment from a few days ago applies to 😄 )

Reply:
now the question makes cents.

Reply:
i see what you did there 👀 

thanks for updating it. approving.